search-api.md

```diff
@@ -23,12 +23,6 @@ After receiving a request, the remote server can respond in one of two ways:
   * If a compatible version (`vX.Z` where `Z>=Y`) is supported by the remote server, it should provide a response using this version.
   * If no appropriate version is supported by the remote server, it should respond with `Not Acceptable (406)`, containing a JSON body with a description of the error. All responses, including this one, should contain a `Content-Type` header with the latest API version supported by the server. This will enable the user to re-submit the request using this version of the API.
```

Comment:
Hmm, say a client submits a request using version 1.0, which *is* supported by the server. Shouldn't the server reply using v1.0 specifications (even if the latest supported version is 2.0 and is totally different and incompatible?). If that is the case, Content-Type of the reply should match the actual content, and not the latest supported version. I don't want to start a versioning hell and fine to require any server to support only one version :), but if multiple versions are supported content-type should be consistent with actual content.

If we want to add ability to discover the list of versions (or latest version) supported by the server I suggest to add a list of supported versions to the "unsupported version number" JSON reply (and possibly specify that version "0.0" should never be supported and can be used - in the unlikely case if anyone ever wants this - to get a guaranteed incorrect version reply with the list of supported versions?